src/main/java/com/github/vkorobkov/jfixturescmd/utils/ExceptionHandler.java

```diff
@@ -0,0 +1,18 @@
+package com.github.vkorobkov.jfixturescmd.utils;
+
+import com.beust.jcommander.ParameterException;
+import lombok.extern.slf4j.Slf4j;
+
+@Slf4j
+public final class ExceptionHandler {
+    private ExceptionHandler() {
+    }
+
+    public static void handleException(Exception e) {
+        if (e instanceof ParameterException) {
+            log.error("Failed to parse command line arguments: " + e.getMessage());
+        } else {
+            log.error(e.getMessage());
```

Comment:
com.beust.jcommander.ParameterException
LoaderException - from core
messages from LoaderException are clear and do not require adding additional info from cmd...

So possible variant: ParameterException as is..
log.error(e.getMessage()) - for LoaderException  
print stackrtace for unexpected?